stories/data.ts

```diff
@@ -3020,6 +3020,1012 @@ export const TABLE: Table = {
       type: 'string',
       name: 'cpu',
     },
+    host: {
```

Comment:
yay! I would love to add more data sets too eventually, besides for just the CPU data set. [vega-datasets](https://github.com/vega/vega-datasets) might be useful.